config/settings.yml

```diff
@@ -44,6 +44,12 @@ goobi:
 
 gis:
   robots_content_root: '/gis_workflow_data/stage'
-  username: 'lyberadmin'
 
 test_folio_instance_hrid: 'a10065784'
+
+# note for purl_url: there is no purl for qa;  sul-h2-qa uses stage ...
+purl_url: 'https://sul-purl-stage.stanford.edu' # from shared_configs for sul-h2-stage
+preassembly:
+  username: 'preassembly'
+  bundle_directory: '/dor/staging/integration-tests/files-reaccessioning-test'
+  hfs_bundle_directory: '/dor/staging/integration-tests/hierarchical-file-test'
```

Comment:
1. these are all the same between qa/stage, put them here instead
2. nest the preassembly related items in a block instead of using `preassembly_bundle_directory` etc